src/pages/DelegationPage/components/DelegationProvidersItem.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <app-button
+    class="delegation-providers-item"
+    scheme="none"
+    size="none"
+    @mouseenter="showDelegateButton"
+    @mouseleave="hideDelegateButton"
```

Comment:
Is your app supposed to work on touch screen devices? 

If yes - everything that happens here on mouse events won't work on those devices